server/backend/api/app/schemas.py

```diff
@@ -73,6 +73,14 @@ class FamUserInfo(BaseModel):
     )
 
 
+class FamUserUpdateResponse(BaseModel):
+    total_users_count: int
+    current_page: int
+    users_count_on_page: int
+    success_user_id_list: List[int]
+    failed_user_id_list: List[int]
```

Comment:
return the success and failed user id list, for example as below:

<img width="299" alt="Screen Shot 2024-07-25 at 3 06 13 PM" src="https://github.com/user-attachments/assets/3ab93ae9-b719-4394-9e6f-eacaba83fe86">